Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -26,7 +26,12 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
         sender.setTitle("Processing...", for: .disabled)
         sender.isEnabled = false
 
-        let request = BTPayPalCheckoutRequest(amount: "4.30")
+        let request = BTPayPalCheckoutRequest(amount: "5.00")
```

Comment:
General q, is there a specific reason we changed the amount here? Not sure if it's needed to test the UPC/line item flow.

Reply:
We get error from gateway, if this amount doesn't match with the amount in lineitem.